tests/osquery/test_api_views.py

```diff
@@ -2401,3 +2418,433 @@ def test_delete_query_permission_denied(self):
         query = self.force_query()
         response = self.delete(reverse("osquery_api:query", args=(query.pk,)))
         self.assertEqual(response.status_code, 403)
+
+    # list pack queries
+
+    def test_get_packqueries_unauthorized(self):
+        response = self.get(reverse("osquery_api:pack_queries"), include_token=False)
+        self.assertEqual(response.status_code, 401)
+
+    def test_get_packqueries_permission_denied(self):
+        response = self.get(reverse("osquery_api:pack_queries"))
+        self.assertEqual(response.status_code, 403)
+
+    def test_get_packqueries_filter_by_pack_id_not_found(self):
+        self.set_permissions("osquery.view_packquery")
+        response = self.get(reverse("osquery_api:pack_queries"), {"pack_id": 9999})
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json(), [])
+
+    def test_get_packqueries_filter_by_slug_not_found(self):
```

Comment:
There is no filter by slug in the view. Here you are only testing that there are not pack queries in the DB.